Portfolio.css

```diff
@@ -1,3 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
 Body {
     font-family: 'Quicksand', sans-serif;
+    font-size: 62.5%;
+    background-color: #fff;
+    color: #5D6063;
+}
+
+.main-nav {
+    background-color: #fff;
+    color: #5D6063;
+    box-shadow: 0 2px 0 rgba(93, 96, 99, .1);
+    width: 100%;
+    font-size: 1rem;
+    padding-bottom: 30px;
+}
+
+.nav-button-left {
+    float: left;
+    position: relative;
+    left: 5px;
+    min-width: 180px;
+}
+
+.nav-button-right {
+    float: right;
+    position: relative;
+    right: 25px;
+    min-width: 180px;
+    text-align: right;
+}
+
+.nav-text {
+    margin-left: 15px;
+}
+
+.main-header-wrapper {
+    max-width: 700px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 80px;
+    padding-bottom: 80px;
+}
+
+.main-content-wrapper {
+    max-width: 700px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 80px;
+    padding-bottom: 30px;
+    overflow: auto;
+}
+
+.blog-wrapper {
+    border-top: 1px solid #EAEDF0;
+    margin-top: 30px;
+}
+
+.article-wrapper {
+    width: 100%
+}
+
+.sf-card {
+    width: 10%;
+    height:10%;
+}
+
+h1 {
+    text-align: center;
+    font-size: 3rem;
+}
+
+h2 {
+    text-align: center;
+    font-size: 2.5rem;
+}
+
+.blog-container {
+    width: 100%;
+}
+
+.article {
+    position: relative;
+    float: left;
+    width: 100%;
+}
+
+.sub-heading {
+    text-align: center;
+    font-size: 1rem;
+}
+
+.sub-heading2 {
+    text-align: center;
+    font-size: 1rem;
+    font-style: italic;
+}
+
+.sub-heading-rule-before {
```

Comment:
Having empty divs to create borders is not best practise. Add the border to an existing element i.e the h2